lib/info.c

```diff
@@ -294,8 +294,16 @@ printKVDataJSON(ktx_uint8_t* pKvd, ktx_uint32_t kvdLen, ktx_uint32_t base_indent
     ktxHashList_Destruct(&kvDataHead);
 }
 
+/**
+ * @internal
+ * @~English
+ * @brief Print the KTX 1/2 file identifier.
+ *
+ * @param [in]  identifier
+ * @param [in]  json specifies if "\x1A" should be escaped as "\u001A" to not break most json tools
+ */
 void
-printIdentifier(const ktx_uint8_t identifier[12])
+printIdentifier(const ktx_uint8_t identifier[12], bool json)
 {
     // Convert identifier for better display.
     uint32_t idlen = 0;
```

Comment:
The indentation is very weird from here until the end of the function. Tabs issue? Please ensure tabs are expanded.

Reply:
Fixed
(It was unrelated to my changes, I try to only effect the relevant code with my changes, fixing these is clang-format territory)